edb/server/protocol/binary.pyx

```diff
@@ -151,13 +151,14 @@ cdef class EdgeConnection(frontend.FrontendConnection):
     def __init__(
         self,
         server,
+        tenant,
```

Comment:
Why pass both `server` and `tenant` and not just `tenant` instead?

Reply:
Mostly because the Postgres protocol may not have a tenant already (TLS handshake is inside the Postgres protocol).